cabal-testsuite/src/Test/Cabal/Monad.hs

```diff
@@ -792,7 +882,7 @@ testUserCabalConfigFile env = testCabalDir env </> "config"
 
 -- | The file where the expected output of the test lives
 testExpectFile :: TestEnv -> FilePath
-testExpectFile env = testSourceDir env </> testName env <.> "out"
+testExpectFile env = testTmpDir env </> testName env <.> "out"
```

Comment:
For the reviewer, this change is crucial, because it allows us to manipulate the output in the running directory, like we do in RejectFutureIndexStates, to replace the REPLACEME string in the output with the returned timestamp.

Reply:
Probably I should note that down in a comment.